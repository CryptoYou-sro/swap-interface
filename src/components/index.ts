export * from './header/header';
export * from './button/button';
export * from './textField/textField';
export * from './swapForm/swapForm';

