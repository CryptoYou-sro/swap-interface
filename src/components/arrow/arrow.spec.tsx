import 'jest-styled-components';
import { render } from '@testing-library/react';
import { Arrow } from '../../components';

describe('Arrow', () => {
	it('should render a default button', () => {
		const { getByText } = render(<Arrow open={true} />);

		expect(getByText(/Arrow/)).toMatchSnapshot();
	});
});
