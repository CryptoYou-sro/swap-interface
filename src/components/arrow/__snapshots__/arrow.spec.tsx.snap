// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow should render a default button 1`] = `
.c0 {
  all: unset;
  cursor: pointer;
  padding-left: 0.875rem;
}

.c0:hover {
  opacity: 0.8;
}

.c0:focus-visible {
  outline-offset: 1px;
  outline: 1px solid #FFF;
}

.c0:active {
  outline: none;
}

.c1 {
  -webkit-transition: all .2s ease-in-out;
  transition: all .2s ease-in-out;
  -webkit-transform: translate(-50%,-50%) rotate(180deg);
  -ms-transform: translate(-50%,-50%) rotate(180deg);
  transform: translate(-50%,-50%) rotate(180deg);
}

<button
  class="c0"
  data-testid="arrow"
>
  <img
    class="c1"
    open=""
    src="arrow-light.png"
  />
</button>
`;
